src/components/AddMoney/components/__tests__/MantecaAddMoney.test.tsx

```
@@ -0,0 +1,149 @@
+
+// records the props MantecaAddMoney hands to the amount step
+let lastInputStepProps: any = null
+jest.mock('@/components/AddMoney/components/InputAmountStep', () => ({
+    __esModule: true,
+    default: (props: any) => {
+        lastInputStepProps = props
+        return <div data-testid="input-amount-step" data-denomination={props.initialDenomination} />
+    },
+}))
+
+// eslint-disable-next-line import/first -- must come after jest.mock
```

ESLint

[error] Definition for rule 'import/first' was not found.

(import/first)